projects/frontend/data-pipelines/gui/projects/data-pipelines/src/lib/components/data-job/pages/details/data-job-details-page.component.spec.ts

```diff
@@ -11,7 +11,7 @@ import { RouterTestingModule } from '@angular/router/testing';
 
 import { BehaviorSubject, of, Subject } from 'rxjs';
 
-import { FORM_STATE, VMWFormState } from '@vdk/shared';
+import { FORM_STATE, VdkFormState } from '@vdk/shared';
```

Comment:
Nit: Renaming this stuff belongs in a different PR. If we have to revert this one for some reason, we'll also be reverting the renaming. Normally I'd be against it, but it doesn't seem likely we'll revert this CI change, so up to you if you want to leave it in.

Reply:
Could be, however the build is actually introduced in this pr, so I would rather have the build to compile the code changed.

Reply:
I considered in case of separation, should the code import fixes be before or after the build script.
In case before, I still cannot verify if the fixes are operational. In case after, the build cannot be merged, due the code fails with compile errors. So I decided to have them both simultaneously.